src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -1081,6 +1100,11 @@ public void serviceChanged(ServiceEvent serviceEvent)
 
     @Override
     public void memberPresenceChanged(ChatRoomMemberPresenceChangeEvent evt)
+    {
+        xmppExecutorPool.execute(() -> memberPresenceChangedInternal(evt));
```

Comment:
These may now execute concurrently or out of order. Do they need to go on a queue?

Reply:
I don't think that matters ...